examples/javascript/main.js

```diff
@@ -1 +1,12 @@
-print("Hello from JerryScript!");
+print("Hello from JerryScript!!!!");
+
+//var n = 0;
+//timer.setInterval(function () { n += 1; print(n, "triggers, time is", timer.now()); }, 1000000);
+
+button = saul.get_one(saul.type.SENSE_BTN);
+led = saul.get_one(saul.type.ACT_SWITCH);
+
+led.write(1);
+
+button.on_threshold(0, function(){led.write(0); return true;}, saul.op.EQ);
```

Comment:
@kaspar030 how about showing some timer function too, for documentation purposes ;)
Something more or less like below, i.e. blink if you press the button?

```
blink = function () {
while (1) {
    value += 1 % 2;
    this.led.write(value);
    t = timer.setTimeout(this.blink, 500000);
   }
}
button.on_threshold(0, this.blink}, saul.op.EQ);
```